packages/nx/src/utils/nx-plugin.ts

```diff
@@ -197,17 +199,27 @@ export function registerPluginTSTranspiler() {
   if (!tsNodeAndPathsRegistered) {
     // nx-ignore-next-line
     const ts: typeof import('typescript') = require('typescript');
```

Comment:
Why are we importing Typescript twice?

Reply:
The line 28 should **_not_** be there. I'll get it removed.